### PR TITLE
ci: speed up circleci workflow turnaround time

### PR DESCRIPTION
**Pull request by devversion**
* See individual commits for changes.

### CODE REVIEW

**Comment by devversion** [comment]
@IgorMinar @gkalpak This should be a good start to work towards making our CI runs shorter. The timing is still not acceptable at all, but this should be a good start. Compared to the ~26min we usually now get.

Also @kara while improving the AIO examples job, I fixed that the SystemJS JIT examples do not run with Ivy. this is now fixed as well, but uncovered two examples that fail. See error in fb1fe61

File: aio/tools/examples/example-boilerplate.js [hunk 1]
```diff
@@ -104,7 +104,12 @@ class ExampleBoilerPlate {
     }
 
     if (ivy) {
-      shelljs.exec(`yarn --cwd ${SHARED_PATH} ivy-ngcc`);
+      // We only need the "fesm5" bundles as the CLI webpack build does not need
+      // any other formats for building and serving. Ngcc currently only updates
+      // the module typings if we specified a "es2015" format, so we need to also
```
Comment (gkalpak) [inline aio/tools/examples/example-boilerplate.js]: _a "es2015"_ --> _an "es2015"_

**Review by gkalpak — APPROVED** [review]
(Still slow 😢) but much better 😃

I would prefer it if the commit messages referred to _"docs examples tests"_ instead of _"AIO example tests"_ (which is not accurate 👨‍🏫) 😇

File: aio/tools/examples/example-boilerplate.js (hunk 1 above)
Comment (devversion) [reply aio/tools/examples/example-boilerplate.js]: Fixed.

**Comment by devversion** [comment]
@gkalpak Yeah. It's still very slow. I'm currently experimenting with running the docs examples concurrently.. we'll see how that goes.

**Pull request closed without merging**